apps/emails-and-messages/src/modules/app-configuration/ui/side-menu.tsx

```diff
@@ -0,0 +1,111 @@
+import { Card, CardContent, CardHeader, Divider } from "@material-ui/core";
+("@material-ui/icons");
+import { DeleteIcon, IconButton, List, ListItem, ListItemCell } from "@saleor/macaw-ui";
+import clsx from "clsx";
+import React from "react";
+
+import { makeStyles } from "@saleor/macaw-ui";
+import { Skeleton } from "@material-ui/lab";
+
+export const useStyles = makeStyles((theme) => ({
+  menu: {
+    height: "fit-content",
+  },
+  clickable: {
+    cursor: "pointer",
+  },
+  selected: {
+    "&&&&::before": {
```

Comment:
what is this? :D

Reply:
Implementation copied from the dashboard, but sure it works as should